Traceback (most recent call last):
  File "weights.py", line 1, in <module>
Traceback (most recent call last):
  File "weights.py", line 1, in <module>
    import numpy as np
ImportError: No module named numpy
    import numpy as np
ImportError: No module named numpy
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[7647,1],0]
  Exit code:    1
--------------------------------------------------------------------------
